### PR TITLE
Add qcomicbook.

### PR DESCRIPTION
**Pull request by greydot**
###### Motivation for this change
Having this program in NixOS.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by samueldr** [comment]
@GrahamcOfBorg build qcomicbook

* * *

Hi! first PR there :) grats!

@GrahamcOfBorg  is our CI pipeline, builds can be queued and thus tested this way (you need permission first for builds). You will soon see the results on each platforms for the build.

File: pkgs/applications/graphics/qcomicbook/default.nix [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    substituteInPlace $out/share/applications/*.desktop \
+      --replace "Exec=qcomicbook" "Exec=$out/bin/qcomicbook"
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/stolowski/QComicBook;
+    description = "Comic book reader in Qt5";
+    license = licenses.gpl2;
+
+    longDescription = ''QComicBook is a viewer for PDF files and comic book archives containing
```
[the remark this answers is not in the document]
Comment (greydot) [reply pkgs/applications/graphics/qcomicbook/default.nix]: Fixed.

**Comment by erikarvstedt** [comment]
I've added a few minor changes that you might want to copy and then it looks about perfect:
https://gist.github.com/erikarvstedt/d3da4c404278b9f50e0485e891bf5e46

Welcome to `nixpkgs` :+1:  :wink:

**Comment by greydot** [comment]
Neat, I didn't know about fetchFromGithub. Thanks!

**Review by matthewbauer — APPROVED** [review]

**Review by xeji — COMMENTED** [review]
Please modify according to nixpkgs [conventions for qt applications](https://nixos.org/nixpkgs/manual/#ssec-qt-applications).

**Comment by greydot** [comment]
@xeji done. Sorry that it took this long.

**Comment by xeji** [comment]
Thank you!
@GrahamcOfBorg build qcomicbook

**Pull request merged**